src/main/kotlin/org/dashevo/dpp/statetransition/StateTransition.kt

```diff
@@ -92,10 +91,16 @@ abstract class StateTransition(var signaturePublicKeyId: Int?,
             }
         }
         signaturePublicKeyId = identityPublicKey.id
-    }
+    }*/
 
+    fun signByPrivateKey(privateKey: ECKey) {
+        val data = serialize(true)
+        val hash = HashUtils.toSha256Hash(data)
 
-    fun verifySignature(publicKey: IdentityPublicKey): Boolean {
+        signature = privateKey.signHash(hash).toBase64Padded()
+    }
+
+    /*fun verifySignature(publicKey: IdentityPublicKey): Boolean {
```

Comment:
Can this be removed?

Reply:
Removed here: https://github.com/dashevo/android-dpp/pull/6/commits/fddc9f82d6f720d931b6ec136e0c0ec1843831ba